QuEST/include/QuEST.h

```diff
@@ -1662,6 +1662,43 @@ int measureWithStats(Qureg qureg, int measureQubit, qreal *outcomeProb);
  */
 Complex calcInnerProduct(Qureg bra, Qureg ket);
 
+/** If both input states \p a and \p b are equal-size density matrices
```

Comment:
Doc should be consistent between argument names (currently `bra` and `ket`, though should become `rho` and `omega`) and symbols in the maths (currently `a` and `b`).

Reply:
Addressed this in acf7f95214544204939c8267381218a72ec98096.